src/database/config/config.js

```diff
@@ -1,12 +1,12 @@
-const config = require('dotenv');
```

Comment:
Is there any reason why you chose to rename this?

Reply:
Yes @timi-codes , I did this for consistency. There are other parts in our codebase that import `dotenv` and use it this way